### PR TITLE
Introduce application settings

### PR DESCRIPTION
**Pull request by yuriidanyliak**
- [x] yaml configuration:  https://github.com/bigbinary/handy, https://github.com/railsconfig/rails_config
- [x] http://railscasts.com/episodes/85-yaml-configuration-file,
- [x] setting env configuration on heroku,
- [x] getting setting from setting object (gem).

### CODE REVIEW

**Comment by yuriidanyliak** [comment]
@sergeyradkevich, created pull request. Deployed: [Heroku](https://immense-peak-9095.herokuapp.com/). Please review.

**Comment by sergeyradkevich** [comment]
@yuriidanyliak, Did you use some different values for production (set in the environment) than the defaults?

**Comment by sergeyradkevich** [comment]
@yuriidanyliak, let's set up [handy](https://github.com/bigbinary/handy) gem in order to get familiar with it. Please create `notices` key (group) and depending on the environment put different messages:
- for development: Rolling RoR
- for production: Beta launch is coming!

Here is the mock up:
settings-notice

Please do not use `Settings` object on a view layer.

**Comment by sergeyradkevich** [comment]
Also, would like to remind to check done items [here](https://github.com/yuriidanyliak/TaDa/pull/10#issue-76384073) (they should be done before the merge).

**Comment by yuriidanyliak** [comment]
@sergeyradkevich, Yes I used another DB user name and auto-generated password.

**Comment by yuriidanyliak** [comment]
@sergeyradkevich, updated. Deployed: [Heroku](https://immense-peak-9095.herokuapp.com/).

File: app/helpers/to_do_lists_helper.rb [hunk 1]
```diff
@@ -1,2 +1,7 @@
 module ToDoListsHelper
+
+  def get_title_notice
+    Settings[:notices]['title-notice']
+  end
```
Comment (sergeyradkevich) [inline app/helpers/to_do_lists_helper.rb]: Isn't it accessible like ordinary call of object's attributes: `Settings.notices.title`?
Also note that no need to repeat `notice` in the name of title.

File: app/views/to_do_lists/index.html.haml [hunk 2]
```diff
@@ -10,6 +10,8 @@
     remote: true,
     'data-toggle': 'modal', 'data-target': '#new_to_do_list_modal'
 
+  .title-notice= get_title_notice#
+
```
Comment (sergeyradkevich) [inline app/views/to_do_lists/index.html.haml]: What does `#` stand for?

**Comment by sergeyradkevich** [comment]
@yuriidanyliak, Reviewed. Please rebase as it cannot be merged currently.

**Comment by sergeyradkevich** [comment]
@yuriidanyliak, I've noticed that auto start of adding of items after list is created doesn't work on tablet (I've used Chrome). Can you please retest on desktop + Chrome + tablet, if available or android cell phone?

**Comment by yuriidanyliak** [comment]
@sergeyradkevich, resolved conflicts, starting to resolve problems with Chrome testing.

**Comment by sergeyradkevich** [comment]
No need to do it within given issue, let's create a new one in the backlog.

**Comment by yuriidanyliak** [comment]
@sergeyradkevich, Ok, I will create such one. But BTW, I have [redeployed](https://immense-peak-9095.herokuapp.com/) and tested the app on the Ubuntu/Chromium and looks like all is working normally. I'll try to test app on the Android phone.

**Comment by yuriidanyliak** [comment]
@sergeyradkevich, would I delete branch `introduce-application-settings`?

**Comment by sergeyradkevich** [comment]
@yuriidanyliak, yes do it, I missed to delete it.

**Pull request merged**